tgui/packages/tgui/interfaces/GravityGen.js

```diff
@@ -0,0 +1,76 @@
+// code\modules\power\gravitygenerator.dm
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, NoticeBox, ProgressBar, Section } from '../components';
+import { Window } from '../layouts';
+
+export const GravityGen = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    charging_state,
+    charge_count,
+    breaker,
+    ext_power,
+  } = data;
+
+  let chargeStatus = state => {
+    if (state > 0) {
+      return (
+        <Box color="average" inline={1}>
+          [ {state === 1 ? "Charging" : "Discharging"} ]
+        </Box>
+      );
+    } else {
+      return (
+        <Box color={ext_power ? 'good' : 'bad'} inline={1}>
+          [ {ext_power ? "Powered" : "Unpowered"} ]
+        </Box>
+      );
+    }
+  };
+
+  let radWarning = state => {
+    if (state > 0) {
+      return (
+        <NoticeBox danger={1} p={2}>
+          <b>WARNING:</b> Radiation Detected!
+        </NoticeBox>
+      );
+    }
+  };
+
+  return (
+    <Window>
+      <Window.Content>
+        {radWarning(charging_state)}
+        <Section>
+          <LabeledList.Item label="Main Breaker">
```

Comment:
This needs to be inside a `LabeledList` - as seen on the screenshot, there is more margin than there should be.

Reply:
Personally I prefer how it looks just by itself. It seems more centered to me.

**Current:**
![old](https://user-images.githubusercontent.com/57483089/105640484-3ac3e280-5e76-11eb-8fcb-a6dd44a99604.png)
**LabeledList:**
![new](https://user-images.githubusercontent.com/57483089/105640489-40b9c380-5e76-11eb-9265-0d4ec9249948.png)
Pretty hard to see the change in the images but the second one is *slightly* higher up than the first.

Reply:
To be honest I'd probably just move the button inside the top-right corner of the Generator Status section. That eliminates one section and keeps everything in the same area.